poseidon/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "poseidon"
+version = "0.1.1"
+description = "Poseidon Hash"
+authors.workspace = true
+repository.workspace = true
+edition.workspace = true
+license = "Apache-2.0"
```

Comment:
Just this field would be enough, may be we can point it to our workspace `Cargo.toml`